spec/rails_helper.rb

```diff
@@ -0,0 +1,22 @@
+require 'spec_helper'
+ENV['RAILS_ENV'] ||= 'test'
+require File.expand_path('../../config/environment', __FILE__)
```

Comment:
I wonder about this change.

Reply:
It is now changed back to how it was before!